Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM nvidia/cuda:11.6.1-base-ubuntu20.04
+
+WORKDIR /app
+
+ENV PYTHON_VERSION=3.8
```

Comment:
This version of python is deprecated. What's the most recent python version we can use?